src/network/networkprotocol.h

```diff
@@ -987,12 +987,12 @@ enum NetProtoCompressionMode {
 };
 
 const static std::string accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
-	"Invalid password",
+	"The password is invalid.",
```

Comment:
I think 'Invalid password' sounds best. Or you could place 'The password you have entered does not match your current password', it is too complicated.